php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2391,7 +2395,14 @@ abstract class NDB_BVL_Instrument extends NDB_Page
                 . ' ';
                 $conditions[] ="t.CommentID NOT LIKE 'DDE_%'";
             }
-            $where = 'WHERE ' . join(' AND ', $conditions);
+            if (is_array($v) && !empty($v)) {
+                $visitString = "('" . implode("','", $v) . "')";
```

Comment:
I think this code is vulnerable to SQL injection since $v came from the user input of the user building the query.

Since the size of $v is variable using prepared statements would probably be annoying, so you should wrap the values in `$DB->quote()` before imploding them.

Something like (untested):

```
$vescaped = array_map($v, function($visit) uses ($db) {
     return $db->quote($visit);
});
$visitString = "('" . implode("','", $vescaped) . "')";
```